pkg/services/mysql/database_arm_template.go

```diff
@@ -24,11 +24,11 @@ var databaseARMTemplateBytes = []byte(`
 			}
 		},
 		"variables": {
-			"DBforMySQLapiVersion": "2017-04-30-preview"
+			"DBforPostgreSQLapiVersion": "2017-12-01"
```

Comment:
Looks like this says "postgres" where it should say "mysql." It's just a variable name, but we should still fix it.